<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page01Intro</class>
 <widget class="QWidget" name="Page01Intro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>351</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Page 01 Intro</string>
  </property>
  <property name="toolTip">
   <string>Page Tool Tip</string>
  </property>
  <property name="statusTip">
   <string>Page Status Tip</string>
  </property>
  <property name="whatsThis">
   <string>Page Whats This</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayoutPage">
     <item row="0" column="0">
      <widget class="QLabel" name="labelTitle">
       <property name="text">
        <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h1 style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:700;&quot;&gt;Projectionist&lt;/span&gt;&lt;/h1&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDetails">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>98</verstretch>
        </sizepolicy>
       </property>
       <property name="statusTip">
        <string>Welcome</string>
       </property>
       <property name="text">
        <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:large; font-weight:700;&quot;&gt;Welcome to Projectionist.&lt;/span&gt;&lt;/h3&gt;&lt;h4 style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:medium; font-weight:700;&quot;&gt;In order to make use of this excellent application, it is first necessary to configure it.&lt;/span&gt;&lt;/h4&gt;&lt;h4 style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:medium; font-weight:700;&quot;&gt;It may also be necessary to add applications to the OS $PATH variable to ensure that they can be found and utilised correctly. This should be fairly easy to do.&lt;/span&gt;&lt;/h4&gt;&lt;h4 style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:medium; font-weight:700;&quot;&gt;Once you are ready to commence the configuration, please click on the Configuration Tab and follow the instructions.&lt;/span&gt;&lt;/h4&gt;&lt;h4 style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:medium; font-weight:700;&quot;&gt;This application can been set to operate in a minimum size of 630 x 500, but can operate just as well or better in large sizes, imcluding full screen.&lt;/span&gt;&lt;/h4&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>-4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelCopyright">
       <property name="text">
        <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot; valign=&quot;bottom&quot;&gt;&lt;span style=&quot; font-size:small; font-weight:700;&quot;&gt;Copyright J2Casa 2023. All Rights Reserved&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/buttonsGlassRound.qrc"/>
  <include location="../../resources/png.qrc"/>
  <include location="../../resources/readfiles.qrc"/>
  <include location="../../resources/splash.qrc"/>
 </resources>
 <connections/>
</ui>
